corehq/apps/cloudcare/static/cloudcare/js/form_entry/entries.js

```diff
@@ -298,13 +298,13 @@ hqDefine("cloudcare/js/form_entry/entries", function () {
                 });
             }
 
-            formEntryUtils.renderMapboxInput(
-                self.entryId,
-                self.geocoderItemCallback,
-                self.geocoderOnClearCallback,
-                initialPageData,
-                self._inputOnKeyDown
-            );
+            formEntryUtils.renderMapboxInput({
+                divId: self.entryId,
+                itemCallback: self.geocoderItemCallback,
+                clearCallBack: self.geocoderOnClearCallback,
+                initialPageData: initialPageData,
```

Comment:
You didn't introduce this, but it caught my eye: why is initialPageData passed as a parameter instead of being imported in form entry utils?

Reply:
good point - seems to just be a relic of the original refactor. Changed to use import [b6d62ac](https://github.com/dimagi/commcare-hq/pull/33533/commits/b6d62acd87583382e0a36dacb00feb76f7ef8f5f)

Reply:
lovely, thank you